tasks/main.yml

```diff
@@ -130,7 +130,7 @@
 
 - name: Copy JBoss EAP
   copy:
-    src: "{{ jboss_eap_artifact_name }}"
+    src: "{{ jboss_eap_artifact_url }}"
```

Comment:
@sabre1041 and @sherl0cks , sorry its my bad - I changed that to test it in local for quicker fixes and forgot to revert it when pushing back . shall i do the change and push it back or its already updated ?